Source/WebKit/UIProcess/API/gtk/WebKitPrintOperation.cpp

```diff
@@ -580,6 +876,8 @@ static GRefPtr<GtkPrinter> printerFromSettingsOrDefault(GtkPrintSettings* settin
  * The #WebKitPrintOperation::finished signal is emitted when the printing
  * operation finishes. If an error occurs while printing the signal
  * #WebKitPrintOperation::failed is emitted before #WebKitPrintOperation::finished.
+ *
+ * Deprecated: 2.46
```

Comment:
It's good to add a little explanation as to why:

```suggestion
 * Deprecated: 2.46. This function does nothing if the app is sandboxed.
```